docs/docs/install-windows.mdx

```diff
@@ -57,6 +58,23 @@ If you want to use a standard theme, you can find them in `~\scoop\apps\oh-my-po
 referencing them as such
 will always keep them compatible with the binary when updating Oh My Posh.
 
+</TabItem>
+<TabItem value="chocolatey">
+
```

Comment:
Adding the admonition here would work:

```markdown
:::info
The chocolatey package is managed by [Curtis Carter][cc]. In case of problems, please create an issue [here][cc-choco].
:::

[cc]: https://github.com/digitalcoyote
[cc-choco]: https://github.com/digitalcoyote/chocolatey-packages/tree/master/oh-my-posh
```